### DISCUSSION TITLE
Actions (3) as property of events

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#115] [Ideas]
Hi Renaud

A button’s click event can execute more then 1 actiion. But you only see the first action and there is no visual clue that there are more actions that will be  executed after a click.
Why not display the actual number of “actions (3)” in the events pane?

Folkert

**Comment by renaudpawlak** [comment]
That's actually a neat idea. I create an issue from here in order to have it done a some point. Thanks :)

**Reply by HFolkertH** [reply]
Ok. Thanks
